examples/react-client/fishjam-chat/src/components/DevicePicker.tsx

```diff
@@ -129,7 +129,20 @@ export function DevicePicker() {
         ) : (
           <Button
             disabled={!hasJoinedRoom}
-            onClick={() => screenShare.startStreaming()}
+            onClick={async () => {
+              try {
+                await screenShare.startStreaming();
+              } catch (error) {
+                if (
+                  error instanceof Error &&
+                  error.name === "NotAllowedError"
+                ) {
+                  // The user closed the screen share window
+                } else {
+                  console.error(error);
+                }
+              }
+            }}
```

Comment:
**nitpick:** please extract the function, it's grown enough
```
try {
    await screenShare.startStreaming();
  } catch (error) {
    if (
      error instanceof Error &&
      error.name === "NotAllowedError"
    ) return;
    console.error(error);
  }
```

Reply:
btw, is `NotAllowedError` only being raised when user close the screenshare window?